inst/examples/modules/app.R

```diff
@@ -16,15 +16,20 @@ selectColumn <- function(input, output, session, df) {
 
   values <- metaReactive2({
     req(input$col)
-    metaExpr(`$`(!!df(), !!input$col))
+    metaExpr({
+      ..(df()) %>%
+        dplyr::pull(!!..(input$col))
+    })
   })
 
   avg <- metaReactive({
-    round(mean(!!values()), 1)
+    ..(values()) %>%
+      mean() %>%
+      round(1)
   })
 
   output$average <- metaRender(renderText, {
-    paste("Average of", !!as.character(input$col), "is", !!avg())
+    paste("Average of", ..(input$col), "is", ..(avg()))
```

Comment:
Interestingly, if you put `..(as.character(input$col))` here, `expandChain()` doesn't do the right thing?

Reply:
Hmm, nevermind, it does the right thing now